spritzle/tests/conftest.py

```diff
@@ -2,45 +2,29 @@
 from pathlib import Path
 import shutil
 
+from aiohttp import web
 import pytest
 
 from spritzle.core import Core
 from spritzle.config import Config
-from spritzle.tests.common import run_until_complete
+from spritzle.main import setup_app
 
-import asyncio
+pytest_plugins = 'aiohttp.pytest_plugin'
 
 
 @pytest.fixture(scope='function')
 def core():
-    # Create a new event loop for each core so that tests can't interfere with eachothers executor pools.
```

Comment:
Do we know if this creates a new event loop for every test? I ran into issues with libtorrent and testing without this (libtorrent segfault randomly). We may need to figure out a way to keep this.

Reply:
Yeah, looks like it to me: https://github.com/aio-libs/aiohttp/blob/master/aiohttp/pytest_plugin.py#L201